app/views/providers/applicants/show.html.erb

```diff
@@ -13,18 +13,19 @@
       ) do |form| %>
 
     <div class="govuk-!-padding-bottom-2"></div>
-
-    <%= form.govuk_text_field :first_name, class: 'govuk-!-width-one-half' %>
-    <%= form.govuk_text_field :last_name, class: 'govuk-!-width-one-half' %>
+    <%= form.govuk_label(:fullname, t('.full_name')) %>
```

Comment:
As I said before, this is problematic. Have a look at the HTML `label` docs:
> The `<label>` tag defines a label for a `<button>`, `<input>`, `<meter>`, `<output>`, `<progress>`, `<select>`, or `<textarea>` element.
In other words the label has to match an input element. There is no `full_name` input element. Full name is entered via TWO fields `first_name` and `last_name`. So you need to treat this as just information provided to the user - almost a heading - that has been styled to look like a label. Therefore I'd suggest:
```
<span class="govuk-fieldset__legend govuk-fieldset__legend--m"><%= t('.full_name') %></span>
```